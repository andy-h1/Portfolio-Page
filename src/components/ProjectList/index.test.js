import React from "react";
import { MemoryRouter } from "react-router-dom";
import { render } from "@testing-library/react";
import { ProjectList } from ".";

describe("Project List Component", () => {
  it("should render", () => {
    const { container } = render(
      <MemoryRouter>
        <ProjectList />
      </MemoryRouter>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
