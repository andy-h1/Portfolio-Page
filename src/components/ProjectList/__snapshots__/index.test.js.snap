// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project List Component should render 1`] = `
<div
  class="sc-AxjAm bfLLc"
>
  <h2
    class="sc-AxirZ dDfKVJ"
  >
    Projects
  </h2>
  <a
    class="sc-AxiKw cCebkB"
    href="/weather"
  >
    <div
      class="sc-AxhCb cVoiXz"
    >
      <h3
        class="sc-AxhUy iAIcNb"
      >
        Weather App
      </h3>
      <p
        class="sc-AxgMl eOWHjm"
      >
        I built this weather app using OpenWeatherMap API to get the current weather data in your location. It has been built with React using styled-components.
      </p>
      <div
        class="sc-AxheI fpnsVe"
      >
        <ul
          class="sc-fzoLsD ihFyfP"
        >
          <li
            class="sc-fzpans eoBJOd"
          >
            react
          </li>
          <li
            class="sc-fzpans eoBJOd"
          >
            styled-components
          </li>
          <li
            class="sc-fzpans eoBJOd"
          >
            jest
          </li>
          <li
            class="sc-fzpans eoBJOd"
          >
            react-testing-library
          </li>
        </ul>
      </div>
      <span
        class="sc-Axmtr hAhsOL"
      >
        <img
          alt="cloudy"
          class="sc-AxmLO VXOhf"
          src="lightCloudsDay.png"
        />
      </span>
    </div>
  </a>
  <a
    class="sc-AxiKw cCebkB"
    href="/Form"
  >
    <div
      class="sc-AxhCb cVoiXz"
    >
      <h3
        class="sc-AxhUy iAIcNb"
      >
        Sign-Up Form
      </h3>
      <p
        class="sc-AxgMl eOWHjm"
      >
        I built this demo sign-up form, hooking the data entries to Firebase's cloud-based database - Firestore. It has been built with React, Firebase and styled-components.
      </p>
      <div
        class="sc-AxheI fpnsVe"
      >
        <ul
          class="sc-fzoLsD ihFyfP"
        >
          <li
            class="sc-fzpans eoBJOd"
          >
            react
          </li>
          <li
            class="sc-fzpans eoBJOd"
          >
            styled-components
          </li>
          <li
            class="sc-fzpans eoBJOd"
          >
            jest
          </li>
          <li
            class="sc-fzpans eoBJOd"
          >
            react-testing-library
          </li>
        </ul>
      </div>
      <span
        class="sc-Axmtr hAhsOL"
      >
        <img
          alt="form"
          class="sc-fzozJi dlipve"
          src="formImage.png"
        />
      </span>
    </div>
  </a>
</div>
`;
